### DISCUSSION TITLE
Can lock prompt on "first broker login" flow on Review Profile

### DISCUSSION THREAD

**Discussion by NtiLan** [keycloak/keycloak#16803] [Q&A]
Hello,
How i can lock some prompt like the username and the email like here : https://i.imgur.com/4MHQWuU.png
Thank for helping ❤

**Comment by NtiLan** [comment]
1 month later and nothing? '-'

**Reply by mrod23** [reply]
Is there a way to remove the "Review profile" button completely?
I have it disabled in the Authentication flow

**Comment by danielFesenmeyer** [comment]
When using the declarative user profile (https://www.keycloak.org/docs/latest/server_admin/index.html#user-profile), you could define the username and email fields as readonly for users. I would expect that those fields are then also readonly on the "Review profile" screen.
